src/components/notification/z-cookiebar/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Component, Prop, h } from "@stencil/core";
+import { ButtonVariantEnum, Theme } from "../../../beans";
+
+@Component({
+  tag: "z-cookiebar",
+  styleUrl: "styles.css",
+  shadow: true
+})
+export class ZCookiebar {
+  /** theme variant (dark | light). default: dark  */
+  @Prop() theme: Theme = Theme.dark;
+  /** cookie policy link url  */
+  @Prop() cookiepolicyurl: string;
+  /** prevent default cookie set action on 'OK' button click (optional) */
+  @Prop() preventcookieset?: boolean;
+  /** hide cookie bar (optional) */
+  @Prop({ mutable: true }) hide?: boolean;
+
+  setAcceptCookie() {
+    if (!this.preventcookieset) {
+      const d = new Date();
+      d.setTime(d.getTime() + 365 * 24 * 60 * 60 * 1000);
+      const expires = "expires=" + d.toUTCString();
+      document.cookie =
+        "cc_cookie_accept" + "=" + "y" + ";" + expires + ";path=/";
+    }
+    this.hide = true;
+  }
+
+  render() {
+    return (
+      <div class={`${this.theme} ${this.hide ? "hidden" : ""}`}>
+        <div class={`wrapper ${this.theme}`}>
+          <div class={`content ${this.theme} `}>
+            <div>
+              {/* <h2 class={`content ${this.theme} `}>Le vostre scelte riguardanti i Cookie su questo sito</h2> */}
+              <h2 class={`content ${this.theme}`}>Su questo sito usiamo i cookies.</h2>
```

Comment:
Wouldn't be better to pass texts as a prop instead of set them inside the component?

Reply:
only for title, I think, description text will be the same forever and ever